app/controllers/register/ConfirmationController.scala

```diff
@@ -64,16 +67,23 @@ class ConfirmationController @Inject()(
                       trn: String,
                       name: String)(implicit request : RegistrationDataRequest[AnyContent]) = {
 
-    userAnswers.get(TrustHaveAUTRPage) match {
-      case Some(true) if isAgent =>
+    val utr = userAnswers.get(TrustHaveAUTRPage)
+    val taxable = userAnswers.isTaxable
+
+    (utr, taxable) match {
+      case (Some(true), true) if isAgent =>
         Future.successful(Ok(existingAgentView(draftId, trn, name)))
-      case Some(true) =>
+      case (Some(true), true) =>
         Future.successful(Ok(existingIndividualView(draftId, trn, name)))
-      case Some(false) if isAgent =>
+      case (Some(false), true) if isAgent =>
+        Future.successful(Ok(newTaxableAgentView(draftId, trn, name)))
+      case (Some(false), true) =>
+        Future.successful(Ok(newTaxableIndividualView(draftId, trn, name)))
+      case (Some(false), false) if isAgent =>
         Future.successful(Ok(newAgentView(draftId, trn, name)))
-      case Some(false) =>
+      case (Some(false), false) =>
         Future.successful(Ok(newIndividualView(draftId, trn, name)))
-      case None =>
+      case (None, _) =>
```

Comment:
@JoshuaH01 I guess these are the non-taxable views? Can you name the package or the files to refer to that please.